src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/IDE/VSSolutionManager.cs

```diff
@@ -37,6 +37,7 @@ namespace NuGet.PackageManagement.VisualStudio
     public sealed class VSSolutionManager : IVsSolutionManager, IVsSelectionEvents
     {
         private static readonly INuGetProjectContext EmptyNuGetProjectContext = new EmptyNuGetProjectContext();
+        private static readonly string VSNuGetClientName = "NuGet VS VSIX";
```

Comment:
nit: could be const :smile:

Reply:
Agreed, make it a const as this will not be referenced outside.